src/read.jl

```diff
@@ -128,7 +128,7 @@ function read_header(io::IO)
         n_signals -= 1
     end
 
-    @assert position(io) == nb_header
+    # @assert position(io) == nb_header
```

Comment:
I don't think we should remove this, since this should always be the case for valid EDF files; the assertion failing indicates that something is wrong.

Reply:
Oh oops I didn't mean to commit this haha